saiph/projection.py

```diff
@@ -16,6 +16,7 @@ def fit(
     nf: Optional[int] = None,
     col_weights: Optional[Dict[str, Union[int, float]]] = None,
     sparse: bool = False,
+    seed: Optional[Union[int, np.random.Generator]] = None,
```

Comment:
Shouldn't seed only be a randomGenerator?

Reply:
`projection.fit` is the main method called from the outside (see `saiph/__init__.py`). I wanted to allow for people that use the `saiph` library as a stand-alone library to use `seed=123`, and not have to worry about internal seeding behavior.